src/main/java/nl/novi/techItEasy/services/CustomUserDetailsService.java

```diff
@@ -0,0 +1,42 @@
+package nl.novi.techItEasy.services;
+
+import nl.novi.techItEasy.dtos.UserDto;
+import nl.novi.techItEasy.models.Authority;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+/*annotatie*/
+@SpringBootApplication
```

Comment:
Hier mag je de @Service annotatie gebruiken. De @SpringBootApplication is alleen voor de Main klasse bedoeld (in dit geval de TechItEasy klasse)